frontend/src/metabase/admin/permissions/selectors/data-permissions/diff.ts

```diff
@@ -1,7 +1,7 @@
 import { createSelector } from "@reduxjs/toolkit";
 import _ from "underscore";
 
-import { diffDataPermissions } from "metabase/admin/permissions/utils/graph";
+import { diffDataPermissions } from "metabase/admin/permissions/utils/graph/permissions-diff";
```

Comment:
why this change? that index file is still intact.

Reply:
Ah I think I moved this then moved it back. I guess I didn't revert it correctly. Will fix.